### PR TITLE
Make update_translations Odoo 11 compatible

### PR DESCRIPTION
**Pull request by p-tombez**

### CODE REVIEW

File: anthem/lyrics/modules.py [hunk 1]
```diff
@@ -24,6 +24,17 @@ def update_translations(ctx, module_list):
...
+        method_name = 'update_translations'
+    else:
+        # Odoo version >= 11.0
+        method_name = '_update_translations'
+
     for module in module_list:
-        ctx.env['ir.module.module'].with_context(overwrite=True).search(
-            [('name', '=', module)]).update_translations()
+        getattr(
+            ir_module.with_context(overwrite=True).search(
```
Comment (guewen) [inline anthem/lyrics/modules.py]: While you are on it, could you add overwrite as parameter in the function?
`overwrite` is not to use lightly as it overrides values entered manually (right?)

Comment (p-tombez) [reply anthem/lyrics/modules.py]: Default value should be ```True``` or ```False``` ?

Comment (grindtildeath) [reply anthem/lyrics/modules.py]: IMO should be `False` :wink:  https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/ir_translation.py#L41

Comment (guewen) [reply anthem/lyrics/modules.py]: overwrite is dangerous so I would use False by default too

**Comment by i-vyshnevska** [comment]
moved in https://github.com/camptocamp/anthem/pull/44

**Comment by sebalix** [comment]
Supersedes by #44, closing

**Pull request closed without merging**